src/app.rs

```diff
@@ -27,6 +27,10 @@ use crate::internal_events::{
     VectorStarted, VectorStopped,
 };
 
+#[cfg(feature = "datadog")]
+static DATADOG_OBSERVABILITY_MESSAGE: &str =
```

Comment:
ITYM `const` instead of `static`

Reply:
Thanks. I moved this back into `try_attach`, so this static is moot.